arches/app/media/js/viewmodels/workflow-step.js

```diff
@@ -209,16 +209,47 @@ define([
 
         this.toggleInformationBox = function() {
             var informationBoxData = self.informationBoxData();
-            var isHidden = informationBoxData['hidden'];
+            var isDisplayed = informationBoxData['displayed'];
 
-            informationBoxData['hidden'] = !isHidden;
+            informationBoxData['displayed'] = !isDisplayed;
             self.informationBoxData(informationBoxData);
 
-            self.setToLocalStorage('informationBoxHidden', !isHidden);
+            self.setMetadataToLocalStorage('informationBoxDisplayed', !isDisplayed);
         };
 
-        this.getInformationBoxHiddenStateFromLocalStorage = function() {
-            return self.getFromLocalStorage('informationBoxHidden')
+        this.getInformationBoxDisplayedStateFromLocalStorage = function() {
+            return self.getMetadataFromLocalStorage('informationBoxDisplayed')
+        };
+
+        this.getMetadataFromLocalStorage = function(key) {
+            var workflowsMetadataLocalStorageData = JSON.parse(localStorage.getItem('workflow-metadata')) || {};
+            var pathArray = window.location.pathname.split('/');
```

Comment:
workflow is a param of workflow-step, and workflowName is on workflow. Can we avoid parsing the location path to get the workflow name?

Reply:
@chrabyrd It looks like it may be parsing the slug from the path rather than the workflow name. The advantage to the slug is that it enforces uniqueness. If the slug was accessible in the step, I think that would be ideal - no parsing needed. Of course, in all likelihood the name would also be unique as well, just not guaranteed (as far as I know).

Reply:
If we need it to have a unique constraint I'd say let's enforce it explicitly at the workflow level and pass it down. I think basing logic off of pathname in this instance is a bit of an anti-pattern, as we don't have that interaction anywhere in workflows, and it's straightforward to use alternative logic.

Reply:
I agree. If the `slug` was available on `config.workflow` then that would be the way to go. However, if using the slug is much trouble at all, it's probably fine to use the workflow name given that having 2 workflows with the same name would be really unlikely.